### ISSUE TITLE
Stylesheet changes to typography

### ISSUE THREAD

**Issue by janagombitova** [akvo/akvo-flow#1869]
@loicsans this issue is to contain all changes made to typography on the dashboard

**Comment by muloem** [comment]
So a series of first comments on the changes related to this issue. 

1. Is it possible to keep the icons for the main navigation menu?  I feel a little lost without them.

<img width="681" alt="screen shot 2016-12-21 at 12 07 28" src="https://cloud.githubusercontent.com/assets/533336/21387072/407c87fe-c776-11e6-9912-1c40e6661b6a.png">

**Comment by muloem** [comment]
2. For some reason the _Add Folder_ and _Create new survey_ buttons are misaligned in my browser. (Chrome)

<img width="1263" alt="screen shot 2016-12-21 at 12 09 55" src="https://cloud.githubusercontent.com/assets/533336/21387134/8c011d84-c776-11e6-9e6d-1b8225f5bb35.png">

**Comment by muloem** [comment]
3. In the same view above you can also see the the little edit icon to the left of the folder name is misaligned.

4. Also in connection with the folder, I feel like there is too much space between the lower border of the folder and the text.  I.e. in terms of vertical alignment of the elements on the folder view, they seems to be upper aligned which looks a little strange with all the space below them.

**Comment by muloem** [comment]
5. Ideally when editing the folder name, the font size of the folder would stay the same as when its saved.

<img width="1263" alt="screen shot 2016-12-21 at 12 13 26" src="https://cloud.githubusercontent.com/assets/533336/21387233/fef4f4fa-c776-11e6-8c95-fb90a5437d09.png">

**Comment by muloem** [comment]
6.  Because the survey meta data appears in bold, it kept drawing my attention away from the survey name, whereas I had the feeling we want to emphasise the survey name more.. maybe having this meta data in smaller font or you have some other way of deemphasising it..?

<img width="1249" alt="screen shot 2016-12-21 at 12 17 07" src="https://cloud.githubusercontent.com/assets/533336/21387387/b12bb2c6-c777-11e6-849c-e0c3046e6304.png">

**Comment by muloem** [comment]
7. Is there a way we can figure out how to minimise the empty spaces in this area?  this way we do not have to scroll very far down to start already editing the forms.  e.g. maybe put the _Privacy type_ and _Language_ dropdown on a single line?  Or showing the survey title only once? E.g. make it editable in the same place in which its shown? (at the top)

<img width="1250" alt="screen shot 2016-12-21 at 12 24 58" src="https://cloud.githubusercontent.com/assets/533336/21387609/ad41c17c-c778-11e6-9fc6-c724d9a14185.png">

**Comment by muloem** [comment]
8. Just like with the surveys, is there a way to minimise the space in between the elements here or even get rid of some of the elements so that the user can arrive at the questions sooner.

<img width="1245" alt="screen shot 2016-12-21 at 12 34 53" src="https://cloud.githubusercontent.com/assets/533336/21387847/e94b37c4-c779-11e6-9c12-c4e2a1e6c377.png">

**Comment by loicsans** [comment]
concerning point 1.
It was a conscious decision to remove those icons, they have no "standalone" meaning. Removing  them really contributed to declutter UI, which is the primary goal of this.

**Comment by loicsans** [comment]
Made fixes for 2, 3, 4, 5, 6 so far.
I agree for 7 and 8 but these will get a fix in an other issue.

**Comment by muloem** [comment]
8. The question mark in the helper looks misaligned + the circle itself

<img width="220" alt="screen shot 2016-12-23 at 13 10 13" src="https://cloud.githubusercontent.com/assets/533336/21453639/5d29960c-c911-11e6-9b3e-1983116733ef.png">

**Comment by muloem** [comment]
9. There is a line cutting the _New form_ button. Its probably a result of latest changes.

<img width="1250" alt="screen shot 2016-12-23 at 13 13 41" src="https://cloud.githubusercontent.com/assets/533336/21453664/c70dcb42-c911-11e6-9dce-5ad9e2b252ea.png">

**Comment by muloem** [comment]
10. The _repeat this group_ text being grey makes it a little hard to read..

<img width="408" alt="screen shot 2016-12-23 at 13 17 10" src="https://cloud.githubusercontent.com/assets/533336/21453712/42a99f92-c912-11e6-8499-de48926f7940.png">

**Comment by muloem** [comment]
is it also possible to style the drop downs, i.e font, colouring?

![Uploading Screen Shot 2016-12-26 at 10.10.25 1.png…]()

**Comment by loicsans** [comment]
It is, but I focus on form elements later on.

**Comment by janagombitova** [comment]
@muloem and @loicsans thank you both for moving forward on this one. I noticed that some of the discussion you had relates to other issues regarding layout or other ones. I think it would be helpful if we clearly know what this issue is tackling as font style, size of items, or also layout to be able to break down the work. What do you think?

**Comment by janagombitova** [comment]
## Notes from testing 

@muloem and @loicsans I reviewed the changes on a Chrome and Safari browser only looking at the typography (not the layout, icons, spacing, etc) and have a few comments: 

### 1) Info text bold
The info texts are in bold. I think they do not need to be bolded. 

![screen shot 2017-01-03 at 11 41 19](https://cloud.githubusercontent.com/assets/12456965/21605456/ae4933a8-d1a9-11e6-8127-ae2fde3d26b3.png)

### 2) Forms, Monitoring, Version, ID and Questions information too small
I find it that the data under the number of forms, Monitoring, version, ID and Questions are too small. These are important information (at least the questions count, monitoring and version number) users rely on regularly and I think the smaller font size in comparison to the current one will take away the importance of this information. 

![screen shot 2017-01-03 at 12 14 22](https://cloud.githubusercontent.com/assets/12456965/21606203/2fcb1cda-d1ae-11e6-950d-4bf487d8ee90.png)

![screen shot 2017-01-03 at 12 14 43](https://cloud.githubusercontent.com/assets/12456965/21606214/3aedadbc-d1ae-11e6-8b5b-7263511600dd.png)

This also applies for Cascades for the number of cascading levels and status 

![screen shot 2017-01-03 at 13 43 05](https://cloud.githubusercontent.com/assets/12456965/21608096/a68d7294-d1ba-11e6-88bf-d8840955378b.png)

### 3) Placeholders too light
All placeholders seem to be too light to me. I find this one hard to read. The placeholders are: 
- survey title, form title and for both descriptions 
- in Options - Code and Option name 
- in Data - all filters 
- in Cascades - Code and Item name (mandatory)
- in Reports - the time filter 
Thank you 

![screen shot 2017-01-03 at 12 17 14](https://cloud.githubusercontent.com/assets/12456965/21606288/ac5418a6-d1ae-11e6-9453-92032d405905.png)

![screen shot 2017-01-03 at 12 37 32](https://cloud.githubusercontent.com/assets/12456965/21606701/7a58b17e-d1b1-11e6-97f6-c669ac018ab2.png)

### 4) Id should be ID
According to our own Jo and Meriam Webster dictionary the short form for identifier is ID. Can you please change it? Cheerios. It is located on: 
- form page 
- Device tab as Device ID 
- in Data > Inspect data - Instance ID, Device ID (in filters and table)
- in Messages > Survey ID

### 5) Save and Loading are now bold
Why are Save and Loading now bold? Is that intentional? Can we find a better way how to highlight both elements? I find the bold a bit hard to read with the current position and white background

### 6) In Translations - Form title and description are bold
I am not sure these two elements deserve to be bold in comparison to the other information on my screen when translating a form. What do you think? 

![screen shot 2017-01-03 at 12 29 38](https://cloud.githubusercontent.com/assets/12456965/21606535/78e0f546-d1b0-11e6-96f6-00f11ff107e7.png)

### 7) In Translations - numbers are bold
Same as above. I think the numbers of questions and options do not need to be bold. 

![screen shot 2017-01-03 at 12 31 06](https://cloud.githubusercontent.com/assets/12456965/21606567/b329082e-d1b0-11e6-8fb0-e8996118e5a2.png)

### 8) In Notifications - Email, Type, Option and Expires are bold 
In Notofications we have Email, Type, Option and Expires shown twice, where the 2nd time it is bold. I like them not bolded out. And yes, I have the feeling I am against bold today :) 

![screen shot 2017-01-03 at 12 33 19](https://cloud.githubusercontent.com/assets/12456965/21606674/4b0991ea-d1b1-11e6-85cf-15b6461aeb6a.png)

### 9) The extra explanatory text can be smaller
In many spots on the dashboard we have an extra sentence explaining stuff. I think this can be smaller in all places. It is located in Assignments, Manual survey transfer, Reports, Bulk upload data, Data cleaning,  

![screen shot 2017-01-03 at 13 08 35](https://cloud.githubusercontent.com/assets/12456965/21607363/e45d49dc-d1b5-11e6-8226-97deed841e85.png)

![screen shot 2017-01-03 at 13 10 17](https://cloud.githubusercontent.com/assets/12456965/21607388/ffd9384c-d1b5-11e6-81d7-748233589758.png)

### 10) font is smaller when editing
In the Inspect data tab when I edit submitted data the font gets smaller in the input field. Can you make it stay the same? 

![screen shot 2017-01-03 at 13 41 06](https://cloud.githubusercontent.com/assets/12456965/21608054/64140d74-d1ba-11e6-8703-b2f97b655d1b.png)

### 11) Charts - answers and Smallest items bold
When the user creates a chart the answers, percentages and total of answers, as well as "Smallest items" are bold and quite large in comparison to the other elements on the page. Can you change that? Thank you 

![screen shot 2017-01-03 at 13 48 45](https://cloud.githubusercontent.com/assets/12456965/21608195/5dbf2b6a-d1bb-11e6-90c4-91d56f6fcdb1.png)

### 12) size of group name and question name 
I personally find the font size of the group name and question name (title) a bit too large in comparison to the other items on the screen. What do you think? 

![screen shot 2017-01-03 at 13 58 04](https://cloud.githubusercontent.com/assets/12456965/21608431/b31bb866-d1bc-11e6-86b7-2c6ae35e083d.png)

**Comment by janagombitova** [comment]
## Notes from testing 

###Notes from previous round 

* [x] 1) Info text bold
* [ ] 2) Forms, Monitoring, Version, ID and Questions information too small - is it intentional that the form: version, ID and questions are smaller than the survey: forms and monitoring? 
* [x] 3) Placeholders too light
* [x] 4) Id should be ID
* [x] 5) Save and Loading are now bold
* [x] 6) In Translations - Form title and description are bold
* [x] 7) In Translations - numbers are bold
* [x] 8) In Notifications - Email, Type, Option and Expires are bold 
* [x] 9) The extra explanatory text can be smaller
* [x] 10) font is smaller when editing
* [ ] 11) Charts - answers and Smallest items bold – Did you make any edits to this one?
* [x] 12) size of group name and question name and Messages

### New notes 

1. Please check points 2 and 11 from previous round 
1. Info text font large. What do you think of also make the font in the info pop-ups smaller? 
1. In Translations – option numbers are bold. Loic I checked that you changes the question order numbers, but I have the feeling the option order numbers are still bold. Can that be? 

@loicsans what I see on the beta1 dashboard is what would go live if all this issue PR is merged right? If so, then I think we need to focus on the style of buttons as many are not different and less visible, if we want this to go out. 

Secondly, I agree on removing the icons from the main menu, but until we do so can you bring the font size back to a smaller one – now the combination looks kind of funky.

**Comment by janagombitova** [comment]
👍  no suggestions for improvement anymore from my side. I think some small cherries will pop up once we improve layout and more, but for now this is good to go. (cross checked on Safari, Chrome, Firefox browsers)

**Comment by janagombitova** [comment]
## Notes from testing on UAT1 (these need to be tacked before we can release) 

### 1) User page not fully showing 

The Users tab does not open fully. 
![user tab](https://cloud.githubusercontent.com/assets/12456965/22502955/977060fc-e86f-11e6-9e9d-fa072f3ef038.gif)

### 2) If the breadcrumb is on 2 lines, survey name not shown 

If the folder-folder breadcrumb is too long and reaches to two lines, the first folder or survey is not fully visible and scrolling does not help. The user cannot fully see the survey name nor open/edit it. 

<img width="1068" alt="screen shot 2017-02-01 at 11 27 55" src="https://cloud.githubusercontent.com/assets/12456965/22503664/0e24a99e-e873-11e6-9e6b-a08bf7f2e5f4.png">

### 3) Save and cancel buttons, not eye catching 

The user has to explicitly hit the Save button to save a question and changes that he made. Previously the button was more "in your face" now I am worried that the users will miss it. 

<img width="1078" alt="screen shot 2017-02-01 at 12 21 48" src="https://cloud.githubusercontent.com/assets/12456965/22504919/08748004-e879-11e6-9faa-a0faa7d45fb8.png">

**Comment by janagombitova** [comment]
## Second round of notes from UAT1 testing 

### 4) the paste button looks odd 

When copy - pasting a question, the button to paste the question looks a bit weird. Can you use the same pattern as with the other buttons? 
<img width="504" alt="screen shot 2017-02-01 at 12 26 33" src="https://cloud.githubusercontent.com/assets/12456965/22505085/d8ecc96c-e879-11e6-928c-4aaa4cee598d.png">

### 5) Long survey name layout broken 

If the survey name is across two lines, the layout is broken in the survey list 

<img width="1251" alt="screen shot 2017-02-01 at 12 47 50" src="https://cloud.githubusercontent.com/assets/12456965/22505619/bd90f1ea-e87c-11e6-9f21-a9e07b389fa9.png">
<img width="737" alt="screen shot 2017-02-01 at 12 49 42" src="https://cloud.githubusercontent.com/assets/12456965/22505659/ee8bf89e-e87c-11e6-90f3-ec982e74e9c5.png">

### 6) Save button for translations 

Same as above with the Save button for questions. I am worried that it is not explicit enough like it is right now. I know we need to do a proper revamp of all buttons, but lets do those all in one issue to align them. For now - can you bring them back to the original state? 

<img width="645" alt="screen shot 2017-02-01 at 12 53 03" src="https://cloud.githubusercontent.com/assets/12456965/22505835/aeb985c8-e87d-11e6-8490-10330ee961b6.png">
<img width="535" alt="screen shot 2017-02-01 at 12 53 12" src="https://cloud.githubusercontent.com/assets/12456965/22505838/b09719fa-e87d-11e6-8635-f85d723626fb.png">

### 7) Buttons back to original state 
@loicsans would it be possible to bring all buttons back to their original look and feel and then we tackle them in a separate issue? Now we have quite the inconsistency where some have a frame, some have a background colour and I think we need to give them all an upgrade in one go. 

### 8) Icon layout in cascades 
When editing the cascade the icon overlays on the text in the buttons making them unreadable 
<img width="636" alt="screen shot 2017-02-01 at 13 19 53" src="https://cloud.githubusercontent.com/assets/12456965/22506533/419d4052-e881-11e6-8aaa-7705f4f09d1b.png">

### 9) Carto map side panel font 

On Carto maps the size of the font for questions is quite large. Is this intentional? Can we make it smaller? 
<img width="488" alt="screen shot 2017-02-01 at 13 29 18" src="https://cloud.githubusercontent.com/assets/12456965/22506818/c4142df6-e882-11e6-899d-bb5db8387353.png">

**Comment by loicsans** [comment]
All points above are addressed expect for the breadcrumb behaviour described in point number 2. This needs to be addressed globally in: https://github.com/akvo/akvo-flow/issues/1852 next.

**Comment by janagombitova** [comment]
## Notes from testing 

### Previous issues 
* [x] 1) User page not fully showing
* [x] 2) If the breadcrumb is on 2 lines, survey name not shown -  #1852
* [x] 3) Save and cancel buttons, not eye catching
* [x] 4) the paste button looks odd
* [x] 5) Long survey name layout broken
* [x] 6) Save button for translations
* [x] 7) Buttons back to original state
* [x] 8) Icon layout in cascades
* [x] 9) Carto map side panel font

### Conclusion

There are two issues still present. but both have their separate GH issues and can be tackled in the next release as they are more improvements that issues. We can consider this ready for release. A great first step 👍
